gradle.properties

```
@@ -0,0 +1,7 @@
+description='Kafka UI'
+version=0.1.0-SNAPSHOT
+group=io.kafbat.ui
+local_node = false
+node_version = 18.17.1
```

```suggestion
node_version = 22.12.0
```
as per aaac4d7c87765303b6b39c2a2a43ee307ce7ab2a

---

pom use 18 version